client/a8c-for-agencies/sections/marketplace/products-overview-v2/product-card/lib/get-woo-product-url.ts

```diff
@@ -0,0 +1,63 @@
+const WOOCOMMERCE_PRODUCT_SLUG_MAP = {
```

Comment:
We will likely move this to the backend to create a scalable solution once we begin supporting over 1,000 third-party extensions.